CI.hs

```diff
@@ -29,15 +29,19 @@ main = do
     renameDirectory (dropExtensions tarball) "ghc-lib-parser"
     removeFile tarball
     removeFile "ghc/ghc-lib-parser.cabal"
+    cmd "git checkout stack.yaml"
```

Comment:
It would be nice to add some comments on what exactly is going on here. E.g., why do you need to add `- ghc` to the end, which section is that even in the `stack.yaml`, why do we only need this for some things, …

Reply:
Right. In fact, the program is a kind of formalization of the `README` which might help in the interim. I'll make an issue for it.